src/helpers/getByAPI.js

```diff
@@ -117,9 +117,32 @@ const getNodeByTestId = (node, testID) => {
 };
 
 export const getByText = (instance: ReactTestInstance) =>
-  function getByTextFn(text: string | RegExp, options?: TextMatchOptions) {
+  function getByTextFn(
+    text: string | RegExp,
+    options: TextMatchOptions = { exact: true }
+  ) {
     try {
-      return instance.find((node) => getNodeByText(node, text, options));
+      const matches = instance.findAll((node) =>
```

Comment:
Hm, seems like we could include this logic in `getAllByText` and reuse here? It's what we do with `ByTestId` queries. Otherwise we get different behavior between `getByText` and `getAllByText`.

Reply:
Yup good point, definitely gonna implement this right away